README.md

```diff
@@ -22,6 +22,7 @@ If you cannot provide a valid website for your company your PR will not be accep
 
 ## A - C
 - [Allied Netlink Technology Pvt. Ltd.](http://ant.com.np/) | Putalisadak, Kathmandu | Internet Service Provider (ISP)
+- [Ambarkaar Software Pvt. Ltd.](http://www.http://compliancequest.com//) | Bakhundol, Lalitpur | Ambarkaar is a subsidiary company of ComplianceQuest. We develope Enterprise Quality Management System in Salesforce.   
```

Comment:
You can remove the first full stop and replace it with a comma. The second full stop can be removed. Thanks! If you are finding it difficult to edit the file, let me know.

Thanks